app/services/forest_liana/filters_parser.rb

```diff
@@ -165,6 +184,9 @@ def is_belongs_to(field)
       field.include?(':')
     end
 
+    def is_smart_field(field)
+    end
```

Comment:
[MISC] Defined only to be overridden right?

Reply:
Oopsy nop

Reply:
Friday morning != Friday afternoon guys 🙏